src/test/kotlin/step2/ParserTest.kt

```diff
@@ -0,0 +1,42 @@
+package step2
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatIllegalArgumentException
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class ParserTest {
+    private val parser = Parser(" ")
+
+    @ParameterizedTest
+    @ValueSource(strings = ["1.1 + 2", "2 * 5", "1 * 10 - 100"])
+    fun `입력값 파싱 정상 케이스`(input: String) {
+        val expected = parser.parse(input)
+        val result = input.split(" ")
```

Comment:
expected, result변수가 반대로 된거같아요.
```suggestion
        val result = parser.parse(input)
       val expected = input.split(" ") 
```

그리고 input.split(" ")로 예상값을 판단하는것보단 실제로 예상값을 파라미터로 전달받는게 좋을거같습니다.
parser의 내부구현이 바뀔수도 있기도 하고, 내부구현에 대해 많은걸 알아야 되니까요.